web_source/cities/europe/albania/berat.md

```diff
@@ -4,12 +4,14 @@ Country: Albania (Europe)
 
 ##### Also known as:
 
-FIXME
+Berati
 
 ### What we would like to see there
 
-FIXME
+Berat Castle, and to admire the view to the city. 
+
+![Berat Castle](https://en.wikipedia.org/wiki/Berat#/media/File:Berat_citadel.jpg)
```

Comment:
I think this link might be broken.

Reply:
Thank you for spotting the broken link, I have hopefully updated with one that works! Please review again.

Reply:
Great image, thanks!